Catalog item registration (#3656)
SkylineCommunications/dataminer-docs: tutorials/Catalog_Tutorials.md

---
uid: Catalog_Tutorials
---

# Catalog tutorials

> As part of dataminer.services, the Catalog module allows Skyline customers and DataMiner Strategic Partners to browse and deploy available DataMiner connectors, packages, Visio drawings, Automation scripts, dashboards, and more.

Reviewer: ```suggestion
As part of dataminer.services, the Catalog module allows DataMiner users and DataMiner Strategic Partners to browse and deploy available DataMiner connectors, packages, Visio drawings, Automation scripts, dashboards, and more.
```
Should no longer use customer I believe

> Using [Catalog Registration API](xref:Register_Catalog_Item), new catalog items can be made available on the Catalog.

Author: ok, adapted

## Tutorials

| Name | Description |
|---|---|

> | [Registering a new connector in the Catalog](xref:Tutorial_Register_Catalog_Item) | Find out how to register a Catalog item using the Catalog API, using the example of registering a connector. |

Reviewer: ```suggestion
| [Registering a new connector in the Catalog](xref:Tutorial_Register_Catalog_Item) | Find out how to register a Catalog item using the Catalog API, following the example of registering a connector. |
```

> | [Registering a new version of a connector in the Catalog](xref:Tutorial_Register_Catalog_Version) | Find out how to register a new version of a Catalog item using the Catalog API, using the example of registering a connector version. |

Reviewer: ```suggestion
| [Registering a new version of a connector in the Catalog](xref:Tutorial_Register_Catalog_Version) | Find out how to register a new version of a Catalog item using the Catalog API, following the example of registering a connector version. |
```
